doc/source/parameters/unresolved-cfd-dem/cfd-dem.rst

```diff
@@ -42,7 +43,15 @@ This subsection includes parameters related to multiphase flow simulations using
     where :math:`\omega` is the angular velocity of the particle.
 
  .. warning:: 
-   We do not recommend using the Magnus lift force. The current model does include any angular momentum dissipation mechanism in the solid-fluid coupling. Using the Magnus force may lead to unphysical results.
+   We do not recommend using the Magnus lift force, especially without enabling `viscous torque`. The Magnus lift force model does include any angular momentum dissipation mechanism in the solid-fluid coupling. Using the Magnus force may lead to unphysical results.
+
+* The ``viscous torque`` parameter controls whether the fluid-particle contact dissipates or not the particles' angular momentum due to viscosity.
+
+..note::
+   By setting ``set viscous torque = true``, the model applied is the one described on the Example 3.6-5 of the book by Bird et al. (2006) [6].
```

Comment:
Put the mathematical expression directly and add the reference instead of just the reference.